src/Utils/OrientationUtils.js

```diff
@@ -186,9 +235,13 @@ export default {
      * @param {number} proj.b - the semiminor radius of the ellipsoid axis
      * @param {number} proj.long0 - the longitude of the central meridian
      *
-     * @return {OrientationFunction} the orientation function
+     * @param {Coordinates} [coordinates]  coordinates the origin of the local East North Up
+     * (ENU) frame
+     * @param {THREE.Quaternion} [target=new THREE.Quaternion()] output Quaternion
+     * @return {FunctionOrQuaternion} The target quaternion if coordinates is defined, otherwise, a function to compute it from coordinates.
     */
-    quaternionToTMerc(proj) {
+    quaternionFromTMerc(proj, coordinates, target = new THREE.Quaternion()) {
```

Comment:
The parameter `proj` may not be useless if we use the `crs coordinates` to find it.